xml/api/resources-2.24.rng

```diff
@@ -37,6 +37,9 @@
             <attribute name="unique"> <data type="boolean" /> </attribute>
             <attribute name="managed"> <data type="boolean" /> </attribute>
             <attribute name="failed"> <data type="boolean" /> </attribute>
+            <optional>
+                <attribute name="description"> <data type="string" /> </attribute>
+            </optional>
```

Comment:
The schema changes look good, but we don't update older schemas -- instead, we modify a new copy (per "General Procedure" in xml/README.md).

I just realized the README is missing a step for the api schema: we also bump PCMK__API_VERSION